Frameworks/UIKit/StarboardXaml/CompositorInterface.mm

```diff
@@ -329,9 +329,10 @@ void SetParams(UIFont* font,
         _centerVertically = centerVertically;
         _lineHeight = [font ascender] - [font descender];
 
-        int mask = [font fontDescriptor].symbolicTraits;
-        _isBold = (mask & UIFontDescriptorTraitBold) > 0;
-        _isItalic = (mask & UIFontDescriptorTraitItalic) > 0;
+        _fontWeight = [font _fontWeight];
+        _fontStretch = [font _fontStretch];
+        _fontStyle = [font _fontStyle];
```

Comment:
got it. sorry about this :(

---
In reply to: [86254978](https://github.com/Microsoft/WinObjC/pull/1302#discussion_r86254978) [](ancestors = 86254978)